oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,7 +64,14 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
               .setSubject(identity.id())
               .setIssuedAt(instant.asDate())
               .setExpiration(new Date(accessToken.expirationTimestamp()))
-              .claim("identity", identityAsJson(identity).toString())
+              .claim("name", identity.name())
+              .claim("email", identity.email())
+              .claim("given_name", identity.givenName())
+              .claim("family_name", identity.familyName())
+              .claim("namespace", identity.claims().get("namespace"))
```

Comment:
You have to iterate over the map as these are custom claims and the library shouldn't know anything about them.